tests/test___init__.py

```diff
@@ -115,53 +116,35 @@ def load(model_name=model_name, layer=layer, region=region, pca_components=pca_c
             model = LayerMappedModel(f"{model_name}-{layer}", activations_model=activations_model)
             model = TemporalIgnore(model)
             model.commit(region, layer)
             return model
 
         return LazyLoad(load)  # lazy-load to avoid loading all models right away
 
-    @pytest.mark.memory_intense
-    def test_alexnet_conv5_IT_temporal(self):
-        model = self.layer_candidate('alexnet', layer='features.12', region='IT', pca_components=1000)
-        score = score_model(model_identifier='alexnet-f12-pca_1000', model=model,
-                            benchmark_identifier='dicarlo.Majaj2015.temporal.IT-pls')
-        assert score.raw.sel(aggregation='center') == approx(0.277449, abs=0.005)
-        assert len(score.raw.raw['time_bin']) == 12
-
 
 @pytest.mark.private_access
 @pytest.mark.memory_intense
 class TestBrainTranslated:
-    @pytest.mark.parametrize(['model_identifier', 'expected_score'], [
-        ('alexnet--pca_1000', .601174),
-        ('alexnet--degrees', .560698),
-        ('alexnet--degrees-pca_1000', .567),
-        ('CORnet-S', .600),
+    @pytest.mark.parametrize(['model_identifier', 'expected_score', 'attach_hook'], [
+        ('alexnet', .59033, True),
+        ('CORnet-S', .600, False),
     ])
-    def test_Majaj2015ITpls(self, model_identifier, expected_score):
+    def test_Majaj2015ITpls(self, model_identifier, expected_score, attach_hook):
         model = brain_translated_pool[model_identifier]
+        if attach_hook:
+            activations_model = model.layer_model._layer_model.activations_model
+            LayerPCA.hook(activations_model, n_components=1000)
+            identifier = activations_model.identifier +  "-pca_1000"
+            activations_model.identifier = identifier
         score = score_model(model_identifier, 'dicarlo.Majaj2015.IT-pls', model=model)
         assert score.raw.sel(aggregation='center') == approx(expected_score, abs=0.005)
 
-    @pytest.mark.parametrize(['identifier_suffix', 'benchmark', 'expected_score', 'time_bins'], [
-        ('--pca_1000', 'dicarlo.Majaj2015.temporal.V4-pls', .281565, 12),
-        ('--pca_1000', 'dicarlo.Majaj2015.temporal.IT-pls', .277449, 12),
-        ('--pca_1000', 'movshon.FreemanZiemba2013.temporal.V1-pls', .120222, 25),
-        ('--pca_1000', 'movshon.FreemanZiemba2013.temporal.V2-pls', .138038, 25),
-    ])
-    def test_alexnet_temporal(self, identifier_suffix, benchmark, expected_score, time_bins):
-        identifier = f'alexnet{identifier_suffix}'
-        model = brain_translated_pool[identifier]
-        score = score_model(identifier, benchmark, model=model)
-        assert score.raw.sel(aggregation='center') == approx(expected_score, abs=0.005)
-        assert len(score.raw.raw['time_bin']) == time_bins
-
     @pytest.mark.parametrize(['model_identifier', 'expected_score'], [
         ('CORnet-S', .25),
-        ('CORnet-R2', .298),
+        ('CORnet-R2', .224),
```

Comment:
this score change is pretty massive, on code that has not changed since it was first added -- are you certain of this?